script.js

```diff
@@ -1 +1,202 @@
+  const completedList = document.getElementById('completedList');
+  const trashButton = document.getElementById('trashButton');
+  const completeButton = document.getElementById('completeButton');
+  const resetButton = document.getElementById('resetButton');
+  let selectedTodos = [];
+
+
+  initializeEventListeners();
+
+function getCurrentDateFormatted() {
+    const today = new Date();
+    const options = { year: 'numeric', month: 'long', day: 'numeric', weekday: 'long' };
+    return today.toLocaleDateString('ko-KR', options);
+  }
+  
+  function toggleSelection(item) {
+    item.classList.toggle('selected');
+    const isSelected = selectedTodos.includes(item);
+  
+    if (isSelected) {
+      selectedTodos = selectedTodos.filter(todoItem => todoItem !== item);
+    } else {
+      selectedTodos.push(item);
+    }
+  }
+  
+  function addTodoItem(text) {
+    const todoItem = document.createElement('li');
+    todoItem.innerText = text;
+    todoList.appendChild(todoItem);
+    updateLocalStorage();
+    updateTodoStatus();
+  }
+  
+  function updateLocalStorage() {
+    localStorage.setItem('todoList', todoList.innerHTML);
+    localStorage.setItem('completedList', completedList.innerHTML);
+  }
+  
+  function loadListFromLocalStorage() {
+    const todoListData = localStorage.getItem('todoList');
+    const completedListData = localStorage.getItem('completedList');
+  
+    if (todoListData) todoList.innerHTML = todoListData;
+    if (completedListData) completedList.innerHTML = completedListData;
+  
+    completedList.querySelectorAll('li').forEach(item => item.classList.remove('selected'));
+    selectedTodos = [];
+  }
+  
+  function setupInputField() {
+    todoInput.addEventListener('input', () => {
+      const trimmedValue = todoInput.value.trim();
+      todoInput.classList.toggle('inputFilled', trimmedValue !== '');
+      if (trimmedValue.length > 20) {
+        todoInput.value = trimmedValue.slice(0, 20);
+      }
+    });
+  }
+  
+  function initializeEventListeners() {
+  
+    todoList.addEventListener('click', event => {
+      if (event.target.tagName === 'LI') {
+        event.target.classList.toggle('clicked');
+        toggleSelection(event.target);
+      }
+    });
+  
+    completedList.addEventListener('click', event => {
+      if (event.target.tagName === 'LI') {
+        event.target.classList.toggle('completed');
+        toggleSelection(event.target);
+      }
+    });
+
+    addButton.addEventListener('click', () => {
+        const text = todoInput.value.trim();
+        if (text) {
+          addTodoItem(text);
+          todoInput.value = '';
+          todoInput.classList.remove('inputFilled');
+        }
+      });
+  
+    trashButton.addEventListener('click', () => {
+      selectedTodos.forEach(todoItem => todoItem.remove());
+      updateLocalStorage();
+      updateTodoStatus();
+      selectedTodos = [];
+      todoInput.value = '';
+      todoInput.classList.remove('inputFilled');
```

Comment:
이 부분이 많이 재사용되는 것 같아서 따로 함수로 빼는 것이 더 가독성이 좋을 것 같아요!
```suggestion
 function clearInputAndSelected() {
    todoInput.value = '';
    todoInput.classList.remove('inputFilled');
    selectedTodos = [];
}
```